topk-js/query.js

```diff
@@ -0,0 +1,9 @@
+const { query } = require("./index");
```

Comment:
wdyt about putting entrypoints in a subfolder? like `/topk-js/entrypoints/...` or some other name but not in the root? root seems cluttered now